cdap-ui/app/directives/log-viewer/log-viewer-store.js

```diff
@@ -15,30 +15,44 @@
  */
 
 var LogViewerStore = (LOGVIEWERSTORE_ACTIONS, Redux, ReduxThunk) => {
-  const startTime = (state = new Date(), action = {}) => {
+  const startTime = (state = null, action = {}) => {
     switch(action.type) {
       case LOGVIEWERSTORE_ACTIONS.START_TIME:
         if(!action.payload.startTime) {
           return state;
         }
         return action.payload.startTime;
       case LOGVIEWERSTORE_ACTIONS.RESET:
-        return new Date();
+        return null;
+      default:
+        return state;
+    }
+  };
+
+  const globalStartTime = (state = null, action = {}) => {
```

Comment:
I am not sure if I understand the difference between startTime and globalStartTime. Probably a short comment in the code and a brief writeup in the PR?